src/component/contentpost/index.tsx

```diff
@@ -53,7 +56,8 @@ function ContentPost(props: ContentPostProps) {
   } = props;
   const { token } = useContext(AuthContext);
 
-  const [showMenu, setShowMenu] = useState(false);
+  const [dropdownVisible, setDropdownVisible, dropdownRootRef] =
```

Comment:
Unrelated, just making the post dropdown close when clicking outside of it (missed this dropdown in previous PRs).